pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -4,11 +4,13 @@ import (
 	"context"
 
 	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/registry"
 )
 
 type AccessControl interface {
+	registry.CanBeDisabled
```

Comment:
Would it make sense to have generic `IsDisabled() bool` instead? As `registry.CanBeDisabled` is quite specific to the registry and comes with certain assumptions (e.g. you call it after init etc).